### PR TITLE
Update on http package  from 0.13.0 to 1.0.0

### PR DESCRIPTION
**Pull request by Emmanuelfrimpong**

### CODE REVIEW

**Comment by domesticmouse** [comment]
@Emmanuelfrimpong please sign the CLA. We can't land this PR without the CLA signed. Thanks!

File: pubspec.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies:
   fixnum: ^1.0.0
   googleapis_auth: ^1.1.0
   meta: ^1.3.0
-  http: ^0.13.0
+  http: ^1.0.0
```
Comment (kevmoo) [inline pubspec.yaml:18]: We want a wide range here to support migration!

```suggestion
  http: '>=0.13.0 <2.0.0'
```

**Review by kevmoo — CHANGES_REQUESTED** [review]
Also please add a changelog entry!

**Comment by devoncarew** [comment]
Looks like the equivalent change landed in https://github.com/grpc/grpc-dart/pull/629.

**Pull request closed without merging**